cli/tools/publish/module_content.rs

```diff
@@ -231,11 +232,12 @@ impl<TSys: FsMetadata + FsRead> ModuleContentProvider<TSys> {
       self.compiler_options_resolver.for_specifier(specifier);
     let jsx_config = compiler_options.jsx_import_source_config()?;
     let transpile_options = &compiler_options.transpile_options()?.transpile;
-    let jsx_runtime = if transpile_options.jsx_automatic {
-      "automatic"
-    } else {
-      "classic"
-    };
+    let jsx_runtime =
+      if transpile_options.jsx_automatic || transpile_options.precompile_jsx {
```

Comment:
Using these options will be less error prone once https://github.com/denoland/deno_ast/pull/313 lands